app/models/account_facility_join.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AccountFacilityJoin < ApplicationRecord
```

Comment:
I really wish we could use `FacilityAccount`… This name is killing me here 😞

Reply:
I would love to rename `FacilityAccount` to `RechargeAccount` (it would be more accurate and avoid some of the confusion in controllers--I'm looking at you `FacilityFacilityAccountsController`), but I feel that's a huge endeavor.